module/files/helper.js

```diff
@@ -1,126 +1,187 @@
 /**
- * name : files.js
- * author : Vishnu
- * created-date : 01-Sep-2023
- * Description :  Files helper.
-*/
+ * name : files/helper.js
+ * author : prajwal
+ * created-date : 25-Apr-2024
+ * Description : All files related helper functionality.Including uploading file
+ * to cloud service.
+ */
 
-const cloudServices = require(GENERICS_FILES_PATH + "/services/cloudStorage")
+// Dependencies
+const { cloudClient } = require(PROJECT_ROOT_DIRECTORY + '/config/cloud-service')
+let cloudStorage = process.env.CLOUD_STORAGE_PROVIDER
 
-module.exports = class FilesHelper {
+/**
+ * FilesHelper
+ * @class
+ */
 
+module.exports = class FilesHelper {
 	/**
-   * Get all signed urls.
-   * @method
-   * @name 								- preSignedUrls
-   * @param {Array} payloadData 		- payload for files data.
-   * @param {String} userId 			- Logged in user id.
-   * @param {String} templateId			- certificateTemplateId.
-   * @returns {Array} 					- consists of all signed urls files.
-   */
+	 * Get downloadable url
+	 * @method
+	 * @name                        - getDownloadableUrl
+	 * @param {filePath}            - File path.
```

Comment:
@Prajwal17Tunerlabs inside {} specify type. not variable name. that you do after

Reply:
got it, have specified the type